spec.bs

```diff
@@ -132,11 +132,21 @@ Each [=environment settings object=] has an <dfn for="environment settings objec
 <p class="note">For import maps, the script never becomes <a spec="html" lt="the script is ready">ready</a> and <a spec="html">the script's script</a> remains null.</p>
 
 <div algorithm>
-  To <dfn export>fetch an import map</dfn> given <var ignore>url</var>, <var ignore>settings object</var>, and <var ignore>options</var>:
+  To <dfn export>fetch an import map</dfn> given |url|, |settings object|, and |options|, run the following steps. This algorithm asynchronously returns a [=string=] or a failure.
+  <p class="note">This algorithm is specified consistently with <a spec="html">fetch a single module script</a> steps 5, 7, 8, 9, 10, and 12.1. Particularly, we enforce CORS to avoid leaking the import map contents that shouldn't be accessed.</p>
+
+  1. Let |request| be a new [=/request=] whose [=request/url=] is |url|, [=request/destination=] is "`script`", [=request/mode=] is "`cors`", [=request/referrer=] is "`client`", and [=request/client=] is |settings object|.
+     <p class="note">Here we use "`script`" as the [=request/destination=], which means the `script-src-elem` CSP directive applies.</p>
+  1. <a spec="html">Set up the module script request</a> given |request| and |options|.
```

Comment:
Is |options| the avenue by which the nonce gets communicated to CSP?

Reply:
I think so. |request|'s nonce metadata is set to |option|'s nonce in https://html.spec.whatwg.org/multipage/webappapis.html#set-up-the-module-script-request.